Remove unused toml files from terrarium test resources

asnytin opened the pull request:
To avoid erroneous mypy runner script invoke

KonstantAnxiety commented:
@asnytin
Looks like `pyproject.toml` it is used in `test_get_compose_path` after all

asnytin commented:
> @asnytin Looks like `pyproject.toml` it is used in `test_get_compose_path` after all

My bad, sorry. Reverted and just added mypy ignore to this `pyproject.toml`.
